indispensable/src/jvmTest/kotlin/at/asitplus/signum/indispensable/EnumConsistencyTests.kt

```diff
@@ -39,6 +39,7 @@ inline fun<reified T: Any> FreeSpec.enumConsistencyTest() {
 }
 
 class EnumConsistencyTests : FreeSpec({
-    enumConsistencyTest<MessageAuthenticationCode>()
-   // enumConsistencyTest<DataIntegrityAlgorithm>()
+    //TODO the tests for these
```

Comment:
TODO? The test for `MessageAuthenticationCode` has been there before

Reply:
i think this is a flaw in the enumconsistencytest itself @iaik-jheher ?